src/libs/auth.ts

```diff
@@ -118,6 +81,10 @@ const sendSignOutMetrics = async (cause: SignOutCause): Promise<void> => {
   });
 };
 
+export const sendRetryMetric = () => {
```

Comment:
Nit: It looks like this is only used once. Is it it worth splitting out into a separate function in a different module?

Reply:
I am doing this as a workaround so that I can use `Ajax().Metrics` from `ajax-common.ts` since otherwise it will have a circular dependency and break tests. If you have any recommendations on other ways to handle this, I would be happy to change it.

Reply:
Fair enough. Yeah, that import cycle is a bear.